packages/react-form/README.md

```diff
@@ -456,6 +456,57 @@ const enabled = useField(false);
 return <Checkbox label="Enabled" {...asChoiceField(enabled)} />;
 ```
 
+#### `asChoiceField()`
```

Comment:
Does it make more sense to just create an `asMutliChoiceField()` and re-use w/e parts of `asChoiceField` we need?